5.2/full/Gemfile

```diff
@@ -99,19 +99,11 @@ gem 'faraday'
 
 # DECLARATIONS
 
-gem 'ibanizator', git: 'https://github.com/wundertax/ibanizator'
```

Comment:
ibanizator, taxger and payday are public, we could leave those. the others would be installed by . the extending containers right?

Reply:
@sudoaza They are not public. We have our private versions.

Reply:
@bryszard ibanizator is public, use anon browsing to https://github.com/wundertax/ibanizator